gui/wxpython/gmodeler/g.gui.gmodeler.html

```diff
@@ -82,10 +83,11 @@ <h3>Main dialog</h3>
 There is also a lower menu bar in the Graphical modeler dialog where one can
 manage model items, visualize commands, add or manage model variables,
 define default values and descriptions. The Python editor dialog window
-allows seeing workflows written in Python code. The rightmost tab of
-the bottom menu is automatically triggered when the model is activated and
-shows all the steps of running GRASS modeler modules. In case of errors
-in the calculation process, it is written at that place.
+allows seeing workflows written in Python code, either as a basic Python
+script, or as a PyWPS script. The rightmost tab of the bottom menu is
+automatically triggered when the model is activated and shows all the steps
+of running GRASS modeler modules. In case of errors in the calculation
+process, it is written at that place.
```

Comment:
they are wrtten? As errors is plural.. or what does "written in that place" refer to?

Reply:
Hard to say as I have just copied that part from the previous version of the docs, but I guess that you're right and it refers to the errors. Reformulated in cb431fcd74cca7750fa3721e283d7b3e2846c061 (and one more little update in 1634cec288369f94099567eb81267532c6c623c6 to make it clearer) - does it make sense now?